### DISCUSSION TITLE
Continuous Recontruction with Helical Projection Data

### DISCUSSION THREAD

**Discussion by MarkatLiu** [astra-toolbox/astra-toolbox#424] [Q&A]
Hi, I'm recontructing the helical projection data using FDK shortscan method. Now the algorithm runs only once based on the projection data within the angle range of 180 degree + 2* FanAngle. My question is how to continuouly run the algorithm with a shift on the projection data so that multiple slices can be reconstructed. Thanks.

**Comment by askorikov** [comment]
There probably won't be a way to do it very efficiently with the current Astra functionality, but you can run a loop reconstructing multiple slices iteratively. Can you provide a minimal code example of what you're doing currently?
